### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by GitHubUser8196** [community/community#124228] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Unfortunately I unlinked my email (from Google account) I am unable to set the email again can anyone please help? 😭🙏

**Comment by Atharvkote** [comment, marked as answer]
Here as shown below you can Login using username tooo?then whats the problem?
![Screenshot_20240517-145240_Chrome](https://github.com/community/community/assets/144655398/6a0a4f77-9562-4fcd-9ac4-d05bd924c8f1)
Also if you have give your biometric in the passkey you can login with it always well and then you can set the primary emails again 
Hope it Help,
And Hey If Your Free, mark this Question as Answered if you want 😁😉

**Comment by davevad93** [comment]
Hi @Saisha16 ,

I didn't quite understand what you mean by unlinking your email address, if you mean linking another email address to your GitHub account follow the steps below:

1. Click on your `Profile picture` in the upper right corner.
2. Go to `Settings`.
3. In the task bar on the left, click on `Emails`.
4. In the `Add email address` field, add your email address.

If you mean linking your email address to gmail, follow these steps:

1. Go to [Gmail](https://mail.google.com/).
2. In the top right, click on `Settings` and then `See all settings`.
3. Click on `Accounts and Import`.
4. In the `Check mail from other accounts` section, click `Add a mail account`.
5. Type the email you want to link and then click `Next`.
6. Select `Link account with Gmail` and then `Next`.
7. Follow the steps on the screen and then click `Next` or `Sign in`.

If you mean accessing your Google Account with another email follow these steps:

1. Go to your Google Account and then click on `Manage your Google Account`.
2. At the top, click on `Personal info`.
3. Under `Contact info`, click on `Email`.
4. Under `Alternate emails`, select `Add alternate email` or `Add other email`.
5. Enter your email address and select `Add`.

**Comment by whatevermykill** [comment]
hello i just enabled my trial accsess to copilot 30 days. but it shows as ended and wants me to pay. can you activate my trial?
